docker-compose.malware-scanner.yml

```diff
@@ -0,0 +1,50 @@
+version: "3.7"
+
+x-logging: &default-logging
+  driver: json-file
+  options:
+    max-size: "10m"
+    max-file: "3"
+
+services:
+
+  clamav:
+    image: clamav/clamav:stable_base
+    container_name: clamav
+    restart: on-failure
+    logging: *default-logging
+    volumes:
+      - ./docker/data/malware-scanner/clamav/defs:/var/lib/clamav
+      - ./docker/malware-scanner/clamd.conf:/etc/clamav/clamd.conf:ro
+    expose:
+      - 3310 # NEVER expose this outside of the local network!
+    deploy:
+      resources:
+        limits:
+          cpus: '0.50'
```

Comment:
can we make it configurable from env variable and default to 0.50 ?

Reply:
Sure, that makes sense.